### ISSUE TITLE
NTR: Vibrio cholerae present on stool culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8103]
New term request Vibrio cholerae present on stool culture (nadea3_190523022101): Serogroup O1 or O139  UMLS: 2020418

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000904
- label: Positive Vibrio cholerae stool culture
- parents: [Positive stool infectious agent test (HP:0034866)](https://hpo.jax.org/app/browse/term/HP:0034866)
- pmid: 
- def: Growth of Vibrio cholerae in a culture derived from a feces sample.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8103
- synonyms: Positive Vibrio cholerae fecal culture(NONE)
